### PR TITLE
WEBAPP-643: Fix display sub categories

### PR DESCRIPTION
**Pull request by Schedulaar**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

**Review by maxammann — APPROVED** [review]

**Comment by Schedulaar** [comment]
@maxammann You can merge this, once you merge WEBAPP-640... My reaction time is probably quite high these days :see_no_evil:

**Review by Taggotty — APPROVED** [review]

**Review by Taggotty — APPROVED** [review]

**Pull request merged**
